src/SourceBuild/tarball/content/smoke-test.sh

```diff
@@ -49,7 +49,6 @@ configuration="Release"
 excludeNonWebTests=false
 excludeWebTests=false
 excludeWebNoHttpsTests=false
-# Re-enable once https://github.com/dotnet/sdk/issues/22734 is fixed.
 excludeWebHttpsTests=true
```

Comment:
Doesn't this need to change?

Reply:
Duh - I don't know what I was thinking.  Thanks for catching.